com.jwestall.Forecast.json

```diff
@@ -0,0 +1,60 @@
+{
+    "id": "com.jwestall.Forecast",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "23.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+    "command": "cosmic-ext-forecast",
+    "finish-args": [
+      "--share=ipc",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--device=dri",
+      "--talk-name=com.system76.CosmicSettingsDaemon",
+      "--filesystem=xdg-config/cosmic:ro"
```

Comment:
why do you need this?

Reply:
I thought they were needed to interact with the cosmic settings daemon. I will remove them when I can get back to my computer.

Reply:
"they". It's only line 14 the question.

Reply:
So line 14 is needed in order for the app to sync with the system theme on the Cosmic desktop. See [Eduado's blog post](https://hachyderm.io/@edfloreshz/112769206427407768) for more details.

Reply:
You have to ask an exception for this https://docs.flathub.org/docs/for-app-authors/linter#exceptions.

Reply:
if you run the linter as described in the submission docs, you can get the error code